lazy_relay.js

```diff
@@ -213,11 +213,16 @@ function onIdentified(err) {
     }
 
     var conn = self.peer.getInConnection(true);
+    if (!conn) {
+        self.logger.warn('onIdentified called on non-existing connection', self.extendLogInfo({}));
```

Comment:
- use `peer.extendLogInfo` too
- also it's regrettable that `PeerDrain#extendLogInfo` isn't called form `Peer#extendLogInfo`, I'd suggest fixing that since I would suspect maybe draining is to blame for this?